kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -420,6 +448,7 @@ def parse(s: str) -> List[T]:
     )
 
     k_kompile_args = ArgumentParser(add_help=False)
+    k_kompile_args.add_argument('--backend', type=str, help='[llvm|haskell]')
```

Comment:
Consider using `pyk.ktool.KompileBackend`:

```suggestion
    k_kompile_args.add_argument('--backend', type=KompileBackend, help='[llvm|haskell]')
```

Then `exec_kompile`, `KEVM.kompile`, etc. can have `backend: KompileBackend` as argument.

Reply:
Done.